src/tests/ftest/soak/harassers.py

```diff
@@ -42,7 +42,41 @@ def test_soak_online_harassers(self):
         :avocado: tags=manual
         :avocado: tags=hw,24
         :avocado: tags=soak,soak_harassers
-        :avocado: tags=SoakHarassers,test_soak_online_harassers
+        :avocado: tags=SoakHarassers,test_soak_online_harassers_12h
+        """
+        test_param = "/run/soak_harassers/"
+        self.run_soak(test_param)
+
+    def test_soak_offline_harassers_24h(self):
+        """Run all soak tests with harassers.
+
+        Test ID: DAOS-2511
+        Test Description: This will create a soak job that runs
+        various offline harassers  defined in the soak yaml
+        This test will run for the time specified in
+        /run/test_timeout.
+
+        :avocado: tags=manual
+        :avocado: tags=hw,24
```

Comment:
Do we really need to have a tag named "24"?

Reply:
yes the 24 is for the wolf perf cluster requiring 24 nodes

Reply:
24 means 24 nodes? I thought 24hr  :-)